include/Methode.tex

```diff
@@ -53,5 +53,12 @@ \subsubsection{Anforderungen an Lösungsansätze, Proof of Concept und MVP}
 \end{itemize}
 
 \subsection{Einschränkungen und Abgrenzungen}
+In diesem Projekt wurde explizit auf eine Implementation mit dem Lagerverwaltungssystem verzichtet, da der Fokus auf der Machbarkeitsstudie lag. Es wurde diese Implementation zwar berücksichtigt, aber sie stellt eine Unbekannte für das Folgeprojekt dar. Weiter wurde explizit die Entwicklung des Prototypen zur Produktionsreife als nicht Teil des Projektes verstanden. Die Referenzimplementation wurde als Validierung der Konzepte und Versuche verstanden, und nicht als Prototyp zur Einbindung in die Umgebung.
+
+Für dieses Projekt wurde auch die Einbindung in die Datenbank des Lagerverwaltungssystem durch eine Momentanaufnahme als Dump abstrahiert, eine dynamische Einbindung ist daher auch Teil des Folgeprojektes.
```

Comment:
Für dieses Projekt wurde auch die Einbindung in die Datenbank des Lagerverwaltungssystem durch eine Momentanaufnahme als Dump abstrahiert, eine dynamische Einbindung ist daher auch Teil des Folgeprojektes.

So nicht ganz korrekt. Es wird nichts abstrahiert, wir greifen gemäss Planung auf eine korrekte Datenbankinstanz zu und nicht auf eine Abstraktion. Wir greifen ja nur nicht auf das Live-System zu, da dazu etwas mehr Zeit beim Kunden vor Ort gebraucht werden würde.

Reply:
En Idee:
Für dieses Projekt wurde der Anschluss an die aktive Datenbank des Lagerverwaltungssystem durch eine lokale Instanz, welche mit einer Momentaufnahmen der realen Datenbank gefüllt ist, ersetzt.